src/main/java/com/mongodb/jdbc/BsonTypeInfo.java

```diff
@@ -23,8 +25,58 @@
     public static final String SYMBOL_TYPE_NAME = "symbol";
     public static final String TIMESTAMP_TYPE_NAME = "timestamp";
     public static final String UNDEFINED_TYPE_NAME = "undefined";
+    public static final String NULL_TYPE_NAME = "null";
```

Comment:
Null is a BSON type and was missing from this class. In addition to the new methods, I also updated the existing ones to handle this type as well. I had it behave the same way as `Undefined`.